### PR TITLE
isort: Sort imports

### PR DESCRIPTION
**Pull request by tony**
To run isort: `isort backend/**/*.py`

`setup.cfg` to should assure consistent config when ran via editors

### CODE REVIEW

**Comment by alexmojaki** [comment]
I'm happy with this since it's just helping tools agree with each other without enforcing anything. We'll have to see if PyCharm has different opinions.

**Comment by tony** [comment]
@alexmojaki Rebased

File: setup.cfg [hunk 1]
```diff
@@ -4,3 +4,15 @@ exclude = .*/,
...
+[isort]
+combine_as_imports= true
+default_section = THIRDPARTY
+include_trailing_comma = true
+multi_line_output = 3
+known_django = django
+known_first_party = main
+split_before_closing_bracket = true
+sections = FUTURE,STDLIB,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+line_length = 79
```
Comment (alexmojaki) [inline setup.cfg:17]:
```suggestion
line_length = 88
```

To match flake8?

**Comment by tony** [comment]
@alexmojaki Added

**Pull request merged**